src/components/Resources/Resources.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
```

Comment:
I think this file can be removed in favor of `src/pages/Resources/Resources.js`, right?